Dockerfile

```diff
@@ -66,15 +67,24 @@ RUN set -ex\
 	; do chgrp -R 0 "$dir" && chmod -R g=u "$dir" ; done\
 	;
 
-# Build-static downloads the static javascript.
-FROM registry.access.redhat.com/ubi8/nodejs-10 AS build-static
+# Build-static downloads the legacy Quay UI.
+FROM registry.access.redhat.com/ubi8/nodejs-10 AS build-legacy-ui
 WORKDIR /opt/app-root/src
 COPY --chown=1001:0 static/  ./static/
 COPY --chown=1001:0 *.json *.js  ./
-RUN set -ex\
-	; npm install --quiet --no-progress --ignore-engines\
-	; npm run --quiet build\
-	;
+RUN set -ex &&\
```

Comment:
`quay.io/projectquay/quay-ui:latest` already has built assets. Why don't we just copy that?